docs/reference/linear-algebra.ipynb

```diff
@@ -138,7 +138,7 @@
     "    data=sc.array(dims=['x','y'], values=np.random.rand(nx, ny)),\n",
     "    coords={'position':sc.geometry.position(x, y, 0.0*sc.units.m)}\n",
     ")\n",
-    "sc.plotting.plot(sensor, projection=\"3d\", positions=\"position\")"
+    "sc.plotting.legacy_plot(sensor, projection=\"3d\", positions=\"position\")"
```

Comment:
I would really like to avoid changing the documentation like this. Can you use the new scatter plot instead? See task in #2864.

Reply:
Then I would have to import `plopp` at the start of the notebook. Is that what we want to do?

Reply:
Well, if not that, how did you plan to make scatter plots in the future?

Reply:
What should we do in the cases where we also need a slider for an additional dimension? e.g. here: https://scipp.github.io/user-guide/coordinate-transformations.html#Sample-data

Reply:
For now I think there is nothing wrong with using the plopp "namespace". We can always add convenience wrappers in scipp itself later.